contracts-periphery/src/WithdrawHook.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+pragma abicoder v2;
+
+import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+import "src/utils/ISwapRouter.sol";
+
+contract WithdrawHook {
+
+  ISwapRouter internal immutable swapRouter;
+  address internal constant WETH9 = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```

Comment:
This needs to be a constructor parameter because it's address is different on every chain

Reply:
Looking again, it seems the contract doesn't use the `WETH9` variable at all?

Reply:
Bumping this

Reply:
Removed in https://github.com/ScopeLift/umbra-protocol/pull/341/commits/b40d4a8da174dc150d24e62c76a20189c2d8cfc2